Move Cloud Security ownership docs under Review and Remediate (#38654)
DataDog/documentation: content/en/security/cloud_security_management/review_remediate/ownership_agent.md

## Supported resource types

The Ownership Agent processes the following resource types:

> **AWS**: `aws_cloudformation_stack`, `aws_cloudfront_distribution`, `aws_cognito_user_pool`, `aws_dynamodb`, `aws_ec2_instance`, `aws_ec2_settings`, `aws_ecr_repository`, `aws_ecs_cluster`, `aws_ecs_service`, `aws_elbv2_load_balancer`, `aws_iam_account`, `aws_iam_policy`, `aws_iam_role`, `aws_iam_role_inline_policy`, `aws_iam_user`, `aws_lambda_function`, `aws_rds_cluster`, `aws_rds_instance`, `aws_s3_bucket`, `aws_secretsmanager_secret`, `aws_security_group`, `aws_subnet`, `aws_vpc`

Reviewer: Moving to definition style was the right direction. Two follow-ups:

- My underlying concern was **density**: the AWS entry is still a single run of 23 comma-separated code spans, which is hard to scan when hunting for one specific type. That's unchanged, just relocated out of the table cell.
- These are standalone **bold paragraphs** rather than list items. The definition-style convention uses list items, and a bulleted list also scans better and chunks more cleanly for AI retrieval. Prefixing each line with `- ` would do it.

Also minor: there's a double blank line above the `## Supported resource types` heading and again after the `Container registries` line.

The Ownership Agent processes the following cloud resource types:

| Cloud provider                    | Resource types |
| --------------------------------- | -------------- |

> | AWS                               | `aws_cloudformation_stack`, `aws_cloudfront_distribution`, `aws_cognito_user_pool`, `aws_dynamodb`, `aws_ec2_instance`, `aws_ec2_settings`, `aws_ecr_repository`, `aws_ecs_cluster`, `aws_ecs_service`, `aws_elbv2_load_balancer`, `aws_iam_account`, `aws_iam_policy`, `aws_iam_role`, `aws_iam_role_inline_policy`, `aws_iam_user`, `aws_lambda_function`, `aws_rds_cluster`, `aws_rds_instance`, `aws_s3_bucket`, `aws_secretsmanager_secret`, `aws_security_group`, `aws_subnet`, `aws_vpc` |

Reviewer: Flipping the columns was the right call. One row per resource type would have been unusable at 27 entries. But the AWS cell is now 23 comma-separated code spans against rows holding one or two, so it wraps into a dense block that's hard to scan when looking for a specific type.

Worth considering a definition-style list instead of a table:

> **AWS**: `aws_cloudformation_stack`, `aws_cloudfront_distribution`, ...
>
> **Azure**: `azure_virtual_machine_instance`

Separately, the **Cloud provider** column header doesn't fit the last row — "Docker, ECR, and other registries" aren't cloud providers. `Source` or `Provider` would cover both cases.

(The list is correctly alphabetized, which helps.)

### Impact on evaluation

Corrections and feedback are vital for tuning the agent. They also affect how the Ownership Agent behaves for that resource going forward.

## Automatic team assignment

By default, high-confidence ownership inferences are applied to the `team` tag on Cloud Security findings. Inferred owners then become available in other Datadog features, including the [Datadog MCP server](/mcp_server/) and automatic notification routing.

> To adjust the confidence threshold or turn off automatic team assignment, use the [ownership settings page](https://app.datadoghq.com/security/configuration/csm/ownership-agent).

Reviewer: The comma and "entirely" fixes look good.

One question from my original comment: this tells readers they can adjust the confidence threshold, and line 86 says high-confidence inferences are applied by default, but **the default threshold value is never stated**. A reader can't tell whether they need to change it or leave it alone. Could you add the default (or say where it's shown in the UI)?

Author: The default threshold value is "high", other values are "medium" and "low"

> Corrections and feedback are vital for tuning the agent to correctly infer ownership in future, and affect how the Ownership Agent behaves for that resource going forward.

Reviewer: Pre-existing, but the line is in this diff's context so it's easy to fix here: "in future" should be "in the future".

```suggestion
Corrections and feedback are vital for tuning the agent to correctly infer ownership in the future, and affect how the Ownership Agent behaves for that resource going forward.
```

Vale also flags this sentence for length (26 words). If you want to split it: "Corrections and feedback are vital for tuning the agent. They also affect how the Ownership Agent behaves for that resource going forward." That drops some redundancy between "infer ownership in the future" and "going forward".

## Automatic team assignment

> By default, high-confidence ownership inferences are applied to the `team` tag on Cloud Security findings. This makes inferred owners available in platform features, through the MCP server, and in automatic notification routing.

Reviewer: Three things in the second sentence:

- The series mixes prepositions (`in` / `through` / `in`), which breaks parallel structure.
- "platform features" is vague — which features?
- "the MCP server" is unspecific and unlinked. Presumably the Datadog MCP server?

```suggestion
By default, high-confidence ownership inferences are applied to the `team` tag on Cloud Security findings. Inferred owners then become available in other Datadog features, including the Datadog MCP server and automatic notification routing.
```

Please confirm the MCP server reference and add a link if there's a docs page for it — I didn't want to guess the URL.

> To adjust the confidence threshold, or to turn off automatic team assignment entirely, use the [ownership settings page](https://app.datadoghq.com/security/configuration/csm/ownership-agent).

Reviewer: The comma incorrectly splits a compound infinitive ("To adjust... or turn off..."), and "entirely" isn't doing any work.

```suggestion
To adjust the confidence threshold or turn off automatic team assignment, use the [ownership settings page](https://app.datadoghq.com/security/configuration/csm/ownership-agent).
```

Separately, a content gap: this section tells readers they can adjust the confidence threshold but never says what the default is, or what "high-confidence" means numerically. The facets table further down establishes that confidence runs 0 to 1, so stating the default value here would let readers decide whether they need to change anything at all.
